osutil/mountprofile_linux.go

```diff
@@ -53,6 +53,15 @@ func LoadMountProfileText(fstab string) (*MountProfile, error) {
 	return ReadMountProfile(strings.NewReader(fstab))
 }
 
+func SaveMountProfileText(p *MountProfile) (string, error) {
```

Comment:
this is not used at all yet afaict

Reply:
This is used in tests in the next chunk.

Reply:
In the next PR you mean? Or somewhere in this PR? If in the next PR, why not simply add it in the next PR when it is used and has the matching tests etc?